Makefile

```diff
@@ -258,21 +258,40 @@ else
 	@echo 'Skipping system tests...'
 endif
 
+start-selenium:
+	@echo "Installing and starting Selenium Server..."
+	@node_modules/selenium-standalone/bin/selenium-standalone install
+	@node_modules/selenium-standalone/bin/selenium-standalone start & echo $$! > SELENIUM_PID
+
+system-test-local: BASEURL?=http://192.168.99.100:9500
+system-test-local: TEST_SERVER?=localhost:4444
+system-test-local: SERVER_NAME?=urth_widgets_integration_test_server
+system-test-local: start-selenium sdist
+	@echo 'Starting system integration tests...'
+	-@docker rm -f $(SERVER_NAME)
+	@OPTIONS=-d SERVER_NAME=$(SERVER_NAME) $(MAKE) server
+	@echo 'Waiting 20 seconds for server to start...'
+	@sleep 20
+	@echo 'Running system integration tests...'
+	@npm run system-test -- --baseurl $(BASEURL) --server $(TEST_SERVER) || (docker rm -f $(SERVER_NAME); -kill `cat SELENIUM_PID`; rm SELENIUM_PID; exit 1)
+	@echo 'System integration tests complete.'
+	@docker rm -f $(SERVER_NAME); kill `cat SELENIUM_PID`; rm SELENIUM_PID
```

Comment:
Line 276 is to kill selenium in case of tests failing, and line 278 kills selenium in case of tests passing